### PR TITLE
SASS-2042: Employment Summary Iteration

### PR DESCRIPTION
**Pull request by joeyymoore**
### Description

- Employment Summary Iteration

https://jira.tools.tax.service.gov.uk/browse/SASS-2042

##### Before Reviewing
 - [x]  Have you pulled the branch down?
 - [x]  Have you assigned yourself to the PR? 
 - [x]  Have you moved the task to “in review” on JIRA? 
 - [x]  Have you checked to ensure all dependencies are up to date? 
 - [x] Have you checked to ensure its been rebased against the current version of master?

##### Whilst Reviewing
 - [x]  Have you run the tests?
 - [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x] Have you checked for merge conflicts or any changes in the current master that may affect the current pull request? i.e. does it need another rebase?
 - [x]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR 
 - [x]  Have you run the tests?
 - [x]  Have you addressed warnings where appropriate? 
 - [x] Have you rebased against the current version of master?

### CODE REVIEW

**Comment by joeyymoore** [comment]
Need to be merged with https://github.com/hmrc/income-tax-employment-frontend/pull/268 and https://github.com/hmrc/income-tax-submission-journey-tests/pull/121

File: src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala [hunk 1]
```diff
@@ -647,7 +647,7 @@ class IncomeTaxSubmissionSimulation extends PerformanceTestRunner {
     getClaimingExpensesPage,
     postClaimingExpensesPage,
     getEmploymentSummaryPage(taxYearEOY),
-    postEmploymentSummaryPage(taxYearEOY)
+    getEmploymentAddNewUserUrl(taxYearEOY)
```
Comment (ina-tu) [inline src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala:650]: I'm not sure if this line is needed

File: src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala [hunk 2]
```diff
@@ -667,7 +667,7 @@ class IncomeTaxSubmissionSimulation extends PerformanceTestRunner {
...
```
Comment (ina-tu) [inline src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala:670]: same here

**Review by ina-tu — CHANGES_REQUESTED** [review]

File: src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala (hunk 2 above)
Comment (joeyymoore) [reply src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala:670]: Same as above

**Review by ina-tu — APPROVED** [review]

**Review by ina-tu — APPROVED** [review]

**Pull request merged**